index.bs

```diff
@@ -170,22 +170,41 @@ User Agent Hints {#http-ua-hints}
-in [[I-D.ietf-httpbis-client-hints]]. The definitions below assume that each [=user agent=] has defined
-a number of properties for itself:
-
-*   <dfn for="user agent" export>brand</dfn> (for example: "cURL", "Edge", "The World's Best Web Browser")
-*   <dfn for="user agent" export>significant version</dfn> (for example: "72", "3", or "28")
-*   <dfn for="user agent" export>full version</dfn> (for example: "72.0.3245.12", "3.14159", or "297.70E04154A")
-*   <dfn for="user agent" export>platform brand</dfn> (for example: "Windows", "iOS", or "AmazingOS")
-*   <dfn for="user agent" export>platform version</dfn> (for example: "NT 6.0", "15", or "17G")
-*   <dfn for="user agent" export>platform architecture</dfn> (for example: "ARM64", or "ia32")
-*   <dfn for="user agent" export>model</dfn> (for example: "", or "Pixel 2 XL")
-*   <dfn for="user agent" export>mobileness</dfn> (for example: ?0 or ?1)
+given [=user agent=], which servers can opt-into receiving via the Client Hints infrastructure
+defined in [[I-D.ietf-httpbis-client-hints]]. The definitions below assume that each [=user agent=]
+has defined a number of properties for itself:
+
+*   <dfn for="user agent" export>brand</dfn> - The [=user agent=]'s commercial name (for example:
+      "cURL", "Edge", "The World's Best Web Browser")
+*   <dfn for="user agent" export>significant version</dfn> - The [=user agent=]'s marketing version,
+      which includes distinguishable web-exposed features (for example: "72", "3", or "12.1")
+*   <dfn for="user agent" export>full version</dfn> - The [=user agent=]'s build version (for
+      example: "72.0.3245.12", "3.14159", or "297.70E04154A")
+*   <dfn for="user agent" export>platform brand</dfn> - The [=user agent=]'s operating system's
+      commercial name. (for example: "Windows", "iOS", or "AmazingOS")
+*   <dfn for="user agent" export>platform version</dfn> - The [=user agent=]'s operating system's
+      version. (for example: "NT 6.0", "15", or "17G")
+*   <dfn for="user agent" export>platform architecture</dfn> - The [=user agent=]'s underlying CPU
+      architecture (for example: "ARM64", or "ia32")
+*   <dfn for="user agent" export>model</dfn> - The [=user agent=]'s device model (for example: "",
+      or "Pixel 2 XL")
+*   <dfn for="user agent" export>mobileness</dfn> - A boolean indicating if the [=user agent=]'s
+      device is a mobile device. (for example: ?0 or ?1)
 
 [=User agents=] SHOULD keep these strings short and to the point, but servers MUST accept arbitrary
 values for each, as they are all values constructed at the [=user agent=]'s whim.
 
+[=User agents=] SHOULD return the empty string or a fictitious value for [=platform architecture=]
+unless the user's platform is one where both the following conditions apply:
+ * Binary download of executables is likely.
+ * Different CPU architectures are likely to require different binary executable resources.
```

Comment:
This is a bit different than the second condition I had in mind, which is that binary downloads for more than one different CPU architectures are likely to be available. So for example, if binary downloads are only commonly found for one CPU architecture, even though others technically exist, that wouldn't meet the condition I was thinking of. I raise this only for consideration. This would be sufficient.

Reply:
That's fair. Adding "likely to be available" to the condition.